BedrockServer.cpp

```diff
@@ -863,13 +863,16 @@ void BedrockServer::runCommand(unique_ptr<BedrockCommand>&& _command, bool isBlo
                 }
             }
 
-            uint64_t conflictLockStartTime = 0;
-            if (lastConflictPage) {
-                conflictLockStartTime = STimeNow();
-            }
-            PageLockGuard pageLock(lastConflictPage);
-            if (lastConflictPage) {
-                SINFO("Waited " << (STimeNow() - conflictLockStartTime) << "us for lock on db page " << lastConflictPage << ".");
+                BedrockCore::AutoTimer *timer = new BedrockCore::AutoTimer(command, BedrockCommand::QUEUE_PAGE_LOCK);
+                uint64_t conflictLockStartTime = 0;
+                if (lastConflictPage) {
+                    conflictLockStartTime = STimeNow();
+                }
+                PageLockGuard pageLock(lastConflictPage);
+                if (lastConflictPage) {
+                    SINFO("Waited " << (STimeNow() - conflictLockStartTime) << "us for lock on db page " << lastConflictPage << ".");
+                }
+                delete timer;
             }
```

Comment:
The indentation in this file is broken starting here, as this should be 4 spaces to the left... I am not going to fix this in this PR 😅